tempesta_fw/http.c

```diff
@@ -2347,6 +2349,9 @@ tfw_http_resp_fwd(TfwHttpResp *resp)
 			TFW_DBG2("%s: Forwarding error: conn=[%p] resp=[%p]\n",
 				 __func__, cli_conn, req->resp);
 			BUG_ON(!req->resp);
+			spin_lock_bh(&cli_conn->seq_qlock);
+			list_del_init(&req->msg.seq_list);
+			spin_unlock_bh(&cli_conn->seq_qlock);
```

Comment:
The list passed to `__tfw_http_resp_fwd()` is not `cli_conn->seq_queue`. The `ret_queue` is cut from `cli_conn->seq_queue`: https://github.com/tempesta-tech/tempesta/blob/774869c747ebf726135b0c210050482c880d20e6/tempesta_fw/http.c#L2305
So the `req` is not a member of `cli_conn->seq_queue`, and the spinlock is not required here.

Reply:
Indeed. Removed.